libs/mintycore/src/main/java/com/solanamobile/mintyfresh/mintycore/usecase/PerformMintUseCase.kt

```diff
@@ -96,9 +96,10 @@ class PerformMintUseCase @Inject constructor(
 
         val transferResult = walletAdapter.transact(sender) {
             val reauth = reauthorize(identityUri, iconUri, identityName, authToken)
```

Comment:
similar comment here. 

`reauthorize` needed? 
transact should handle the auth vs reauth internally right.

Reply:
yep ty, cleaned all this up